File: twist.cpp
#include "Arduino.h"

#include "Servo.h"

int servoPin = 9;
Servo twist;

void setup() {
  Serial.begin(9600);
  twist.attach(servoPin);
}

void loop() {
  if (Serial.available()) {
    char command = Serial.read();
    if (command == 't') {
      twist.write(0);
      delay(1000);
      twist.write(180);
      delay(3200);
    }
  }
}

File: twist.py
import serial
import time

# Replace with your Arduino port (e.g., 'COM3' on Windows or '/dev/ttyUSB0' on Linux/Mac)
arduino = serial.Serial('COM6', 9600, timeout=1)
time.sleep(2)  # Wait for Arduino to reset

# Send the command to trigger servo twist
arduino.write(b't')

arduino.close()